.github/workflows/kind.yml

```diff
@@ -7,6 +7,10 @@ on:
         required: false
         type: string
         default: 'false'
+      caller_repo:
+        required: false
+        type: string
+        default: 'stolostron'
```

Comment:
You should be able to pull this in dynamically rather than having to set it. (Check out the `github.event` object similar to the `RELEASE_BRANCH`  in the `env`.)